### PR TITLE
feat(ida-86): missing column values now will render with unknown value

### PR DESCRIPTION
**Pull request by maqboolkhan**
Missing columns values was rendering with `?`! Now will render with `unknown` column

### CODE REVIEW

File: backend-server/src/main/java/org/dice/ida/util/DbUtils.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.dice.ida.util;
+
+public class DbUtils {
+	public static String manageNullValues(String value) {
+		return value.equals("?") ? "<ida-unk>" : value;
```
Comment (nikit91) [inline backend-server/src/main/java/org/dice/ida/util/DbUtils.java:5]: you should handle a range of values here. Including blank spaces, "?" etc. Also, please put these literals in the constants file.

**Review by nikit91 — CHANGES_REQUESTED** [review]

**Comment by nikit91** [comment]
@nandeeshp24 Could you please implement the frontend logic for handling unknown values in this branch?

File: backend-server/src/main/java/org/dice/ida/util/DbUtils.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.dice.ida.util;
+
+import org.dice.ida.constant.IDAConst;
+
+public class DbUtils {
+	public static String manageNullValues(String value) {
+		return (value == null || ValidatorUtil.isStringEmpty(value) || value.equals("?")) ? IDAConst.NULL_VALUE_IDENTIFIER : value;
```
Comment (nikit91) [inline backend-server/src/main/java/org/dice/ida/util/DbUtils.java:7]: "?" should be in constants. All string literals that can be changed later on should be in constants

**Comment by maqboolkhan** [comment]
@sourabhpoddar404 will take care of Date type column as we talked over`Element`.

**Review by nikit91 — APPROVED** [review]
lgtm

**Pull request merged**